.github/workflows/README.md

```diff
@@ -0,0 +1,55 @@
+# GitHub Actions Workflows
+
+## Setup
+- workloads run using [GitHub self-hosted runners](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/about-self-hosted-runners)
+- project admins maintain a private Google Compute Engine VM for running tests
+  - VM should be at least n1-standard-4 with 50GB persistent disk
+  - instructions for setting up the VM can be found in repo settings under "Actions"
+  - to set up dependencies, run the following commands:
+    ```
+    # install kubectl
+    sudo apt-get install kind kubectl
+    # install kind
+    curl -Lo ./kind "https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-$(uname)-amd64"
+    chmod +x ./kind
+    sudo mv ./kind /usr/local/bin
+    # install skaffold
+    curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/latest/skaffold-linux-amd64
```

Comment:
you can perhaps do `&& \` for the related commands and separate them into code blocks for better readability.

Reply:
done